backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/common/utils/ansible/AnsibleCommandConfig.java

```diff
@@ -307,15 +295,15 @@ private File setupAnsibleRunnerExecutionDir() throws IOException {
     }
 
     private void createHostFile(File inventory) {
-        File hostsFile = new File(String.format("%1$s/hosts", inventory));
-        if (CollectionUtils.isNotEmpty(hostnames())) {
-            try {
-                hostsFile.createNewFile();
-                Files.write(hostsFile.toPath(), StringUtils.join(hostnames(), System.lineSeparator()).getBytes());
-            } catch (IOException ex) {
-                throw new AnsibleRunnerCallException(
-                    String.format("Failed to create inventory file '%s':", hostsFile.toString()), ex);
-            }
+        File hostFile = new File(String.format("%1$s/hosts", inventory));
+        try {
+            hostFile.createNewFile();
+            Files.write(hostFile.toPath(),
+                    String.format(this.host.getHostName() + " ansible_port=%1$s", this.host.getSshPort()).getBytes());
```

Comment:
Can `this.host` be null? I think so, looking to `AnsibleExecutor.createAuditLogable()` below.